modules/codegen-plugin/src/sbt-test/codegen-plugin/aws-newtype-flatten/build.sbt

```diff
@@ -0,0 +1,17 @@
+lazy val root = (project in file("."))
+  .enablePlugins(Smithy4sCodegenPlugin)
+  .settings(
+    scalaVersion := "2.13.10",
+    Compile / smithy4sAllowedNamespaces := List(
```

Comment:
if you add the `smithy4s-aws-kernel` library dependency (instead of core), then yes. We treat `aws` as a [reserved namespace](https://github.com/disneystreaming/smithy4s/blob/series/0.18/modules/codegen/src/smithy4s/codegen/internals/CodegenImpl.scala#L126) and disable its generation by default.